package.json

```diff
@@ -294,8 +295,65 @@
     "watchify": "^3.11.1",
     "webpack": "^4.41.6"
   },
+  "allowScripts": {
+    "node-sass": "4.x.x",
+    "optipng-bin": "6.0.0",
+    "gifsicle": "4.x.x",
+    "jpegtran-bin": "4.0.0",
+    "scrypt": "6.x.x",
+    "chromedriver": "79.0.0",
+    "geckodriver": "1.19.1",
+    "@sentry/cli": "1.58.0",
+    "weak": false,
+    "core-js": false,
+    "core-js-pure": false,
+    "fsevents": false,
+    "keccak": false,
+    "secp256k1": false,
+    "sha3": false,
+    "bufferutil": false,
+    "utf-8-validate": false,
+    "web3": false,
+    "electron": false,
+    "ejs": false,
+    "sc-uws": false,
+    "sqlite3": false,
+    "leveldown": false,
+    "ursa-optional": false,
+    "tiny-secp256k1": false,
+    "gc-stats": false
+  },
```

Comment:
Why are we doing `allowScripts` here and `lavamoat.allowScripts` starting on L330?

Also, related but non-blocking question, could (should?) this move to some sort of config file?